samples/SingleProjectSandbox/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think you committed this file by mistake?

Reply:
Argh, sorry about that, I fixed it.